.github/workflows/release.yml

```diff
@@ -46,9 +37,57 @@ jobs:
         run: |
           python -m build --sdist --outdir dist
 
+      - name: Upload dist directory
+        uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1
+        with:
+          name: parsons-dist
+          path: dist/
+
+  test:
+    needs: build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
+    runs-on: ${{ matrix.os }}
```

Comment:
i don't thing we really need a matrix here, just ubuntu should be fine. but it's up to you.

Reply:
I'm not personally familiar enough with Python packaging to know whether there are realistic scenarios in which a package would install on one operating system and not another, so I included it out of an abundance of caution.

Reply:
We don't cut releases a ton so I think running the extra matrix jobs should not be a big deal.